rasterview.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="image/pwg-raster">
+    <comment>PWG raster image</comment>
+    <glob pattern="*.pwg"/>
```

Comment:
Also need .ras for CUPS raster.

Reply:
see earlier comment: in shared-mime-info, `.ras` is associated with application/x-cmu-raster